QUICK_START.md

```diff
@@ -21,7 +21,7 @@ This document provides quick access to several different ways to set up CORTX fo
 1. Build entire CORTX into OVA (virtual machine image): TODO
     1. Run it on a single node: TODO 
     1. Run it on a cluster: TODO
-1. LR1 Installation Guide: [TODO]
+1. LR1 Installation Guide: [LINK](https://github.com/Seagate/cortx/blob/main/doc/Build_a_community_version_using_the_same_hardware_as_in_Lyve_Rack) 
```

Comment:
Link is broken so please fix

Reply:
Also, it should be a relative link not an absolute one since it is in the same repo.